js/NewsController.js

```diff
@@ -0,0 +1,130 @@
+import {
+    constants
```

Comment:
If there is one import this statement can be written in one line.

Reply:
Changed the code to one line